lib/rules/web/admin_console/4.1/route.xyaml

```diff
@@ -93,3 +93,69 @@ upload_destination_ca_crt_file:
     visible: false
     type: filefield
     op: set <destination_ca_certificate>
+filter_and_show_accepted_routes:
```

Comment:
How about `disable_rejected_box` ?

Reply:
talked in person and no changes needed anymore